feat(content): Repeatable Culture Conversation for Three Card Monte Game (#11773)
endless-sky/endless-sky: data/human/culture conversations.txt

mission "Three Card Monte"
	minor
	repeat
	# invisible
	non-blocking
	"offer precedence" 10
	to offer
		random < 20
		random < 2 + 5 * "flagship planet attribute: tourism" + 5 * "flagship planet attribute: urban" + 10 * "flagship planet attribute: pirate" - "Three Card Monte: declined"
		credits > 1000
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Pirate"
		not attributes "deep" "station"
	on offer
		conversation
			`${three card monte opening}"The game's simple. Stake 1,000 credits and find the ace to double your money."`
			`	The dealer tips a weathered hat, "Always a pleasure to see another captain of fortune." His gaze never leaves your wallet`
				to display
					"reputation: Pirate" > 100
			action
				"monte: games played" = 0
				"monte: not scam chance" = "reputation: Pirate" / 10 + 50
				"monte: not scam chance" >?= 10 # 10% floor, so always at least a 60% of not being scammed first round, meaning the player always comes out ahead in the first game
			label newgame
			`	The dealer waits expectantly.`
			choice
				`	(Put down 1,000 credits to play the game.)`
					to display

> 						credits > 1000

Reviewer: Maybe add a choice where you can say that you don't have enough money if credits < 1000?

		"and find your path blocked by a tight ring of locals listening as a nimble-fingered dealer swaps three cards across a board. "
		"when a sharp laugh draws your attention to a small card game on a crate. "
		"and see a tight cluster of faces leaning in around a makeshift table as quick hands shuffle three tired cards. "
	word
		"The dealer looks up, grins, and offers a simple challenge: "
		"The dealer glances up, eyebrows arched, as he spreads his arms over the cards in invitation: "
		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you the step closer: "
		"The dealer finally notices you, his grin widening as he gestures grandly at the three waiting cards: "

mission "Three Card Monte"
	minor
	repeat
	# invisible
	non-blocking
	"offer precedence" 10
	to offer
		random < 100
	#	random < 2 + 5 * "flagship planet attribute: tourism" + 5 * "flagship planet attribute: urban" + 10 * "flagship planet attribute: pirate" - "Three Card Monte: declined"
		credits > 1000
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Pirate"
		# not attributes "deep" "station"
	on offer
		conversation

> 			`${three card monte opening}"The game's simple. Stake 1,000 credits and find the ace to double your money."`

Author: Rearranged the next line to remove the quotation mark issue.

> 			`	"Always a pleasure to see another captain of fortune," the dealer says, tipping a weathered hat. His gaze never leaves your credits.`

Author: I figured by looking at lumps in your clothing for visual cues, but perhaps the phrasing doesn't work very well. Would changing this to "pockets" or something work better?

> 			`	"Always a pleasure to see another captain of fortune," the dealer says, tipping a weathered hat. His gaze never leaves your credits.`

Reviewer: Maybe wallet? If you think that wouldn't work, pockets is fine.

		"and find your path blocked by a tight ring of locals listening as a nimble-fingered dealer swaps three cards across a board. "
		"when a sharp laugh draws your attention to a small card game on a crate. "
		"and see a tight cluster of faces leaning in around a makeshift table as quick hands shuffle three tired cards. "
	word
		"The dealer looks up, grins, and offers a simple challenge: "
		"The dealer glances up, eyebrows arched, as he spreads his arms over the cards in invitation: "
		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you to step closer: "
		"The dealer finally notices you, his grin widening as he gestures grandly at the three waiting cards: "

mission "Three Card Monte"
	minor
	repeat
	# invisible
	non-blocking
	"offer precedence" 10
	to offer
		random < 20
		random < 2 + 5 * "flagship planet attribute: tourism" + 5 * "flagship planet attribute: urban" + 10 * "flagship planet attribute: pirate" - "Three Card Monte: declined"
		credits > 1000
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Pirate"

> 		not attributes "deep" "station"

Reviewer 2: This would allow this mission to offer on planets like Hermes, a restricted medical world.

Author: Hmmm, I'm a bit torn around this since we could restrict the mission to something like `attributes "tourism" "urban" "pirate"`, though cutting down on the systems it can occur on is a bit antithetical to the goal of filling human space with these backstop missions. 

Taking the example of Hermes, while it does require a government permit to settle on it, the planet attributes (`medical paradise research`) and other missions that can occur on it don't seem like they would completely preclude something like three card monte.

Missions like "Paradise Cleaner Incident" and "News Reporter - Paradise" show that the planet isn't _that_ restricted, since you still have rich traveling businessperson families, poor working families, and reporters flanked by camera drones at the spaceport. To me, that's enough for it to be plausible for something analogous to the child in "Paradise Cleaner Incident" to exist - there's a family where someone has an official job & permit to live there, while another family member is unemployed and runs the scam (while almost certainly putting their family at risk by doing so).

I guess four options are:
1. Leave as is.
2. Add one or two more `not attributes` like the below, though that cuts out a few planets that have tourism or urban attributes.
```suggestion
		not attributes "deep" "station" "military" "research"
```
3. Cut down to _only_ occur if a planet has some positive attributes
```suggestion
		attributes "tourism" "urban" "pirate" "rich"
```

Any preference among those three or additional ideas around what would be the best solution?

Reviewer: Personally I'd prefer the first option; offering on Hermes doesn't seem like a problem.

	on offer
		conversation
			`${three card monte opening}"The game's simple. Stake 1,000 credits and find the ace to double your money."`

> 			`	The dealer tips a weathered hat. "Always a pleasure to see another captain of fortune." His gaze never leaves your wallet.`

Reviewer 2: Wouldn't it make more sense for this to happen before the dealer introduces the game?

				to display
					"reputation: Pirate" > 100
			action

> 				"monte: games played" = 0

Reviewer 2: This should already be the case after the mission is finished.
```suggestion
```

Author: Nice catch.

> 				"monte: not scam chance" = "reputation: Pirate" / 10 + 50

Reviewer 2: Wouldn't it be more intuitive to have the "not scam chance" be the actual chance of not getting a scam, instead of being 50 lower than it?
```suggestion
				"monte: not scam chance" = "reputation: Pirate" / 10 + 100
```

				"monte: not scam chance" >?= 10 # 10% floor, so always at least a 60% of not being scammed first round, meaning the player always comes out ahead in the first game

> 			label newgame

Reviewer 2: ```suggestion
			label "new game"
```

			`	The dealer waits expectantly.`
			choice
				`	(Put down 1,000 credits to play the game.)`
					to display
						credits > 1000

> 				`	(Pat your empty pockets. Can't play today.)`

Reviewer 2: Non-verbal actions are in first-person.
```suggestion
				`	(Pat my empty pockets. Can't play today.)`
```

					decline
					to display
						credits < 1000
				`	(Skip the game.)`
					decline
			action
				"monte: multiplication" = 1 + "roll: 2"
				"monte: addition" = "roll: 3"
				"monte: card spot" = ( "monte: multiplication" + "monte: addition" ) % 3
				"monte: step" = 1
				clear "monte: scammed"
				payment -1000
			`	You drop a 1,000 credit chip to play. "All you need to do is find the ace," the dealer says, flashing the ace faceup towards you, then setting it down between two other facedown cards in the center. He starts shuffling with a practiced twist.`
				goto begin
			label nextdecision
			branch ending
				"monte: step" >= 3 + "monte: games played" * 2
			action
				"monte: step" ++
				"monte: multiplication" = 1 + "roll: 2"
				"monte: addition" = "roll: 3"
				"monte: card spot" = ( "monte: multiplication" * "monte: card spot" + "monte: addition" ) % 3
			label begin
			branch swap1
				and
					"monte: multiplication" == 1
					"monte: addition" == 0
			branch swap2
				and
					"monte: multiplication" == 1
					"monte: addition" == 1
			branch swap3
				and
					"monte: multiplication" == 1
					"monte: addition" == 2

> 			branch swap4

Reviewer 2: ```suggestion
			branch "swap 4"
```

				and
					"monte: multiplication" == 2
					"monte: addition" == 0

> 			branch swap5

Reviewer 2: ```suggestion
			branch "swap 5"
```

				and
					"monte: multiplication" == 2
					"monte: addition" == 1

> 			# swap6 mult 2 add 2, swapping left & right

Reviewer 2: ```suggestion
			# swap 6: mult 2 add 2, swapping left & right
```

			`	The dealer swaps the left and right cards, trading their places in a practiced motion.`
				goto nextdecision

> 			label swap1 # no change

Reviewer 2: ```suggestion
			label "swap 1" # no change
```

			`	The dealer taps the left card, a mock swap leaving the layout untouched.`

> 				goto nextdecision

Reviewer 2: ```suggestion
				goto "next decision"
```

> 			label swap2 # swap clockwise

Reviewer 2: ```suggestion
			label "swap 2" # swap clockwise
```

			`	The dealer rotates the whole set of cards to the right, the rightmost card rolling around to the left.`

> 				goto nextdecision

Reviewer 2: ```suggestion
				goto "next decision"
```

			label swap3 # swap counterclockwise
			`	The dealer shifts the trio one step to the left, the leftmost card ending up on the right.`

> 				goto nextdecision

Reviewer 2: ```suggestion
				goto "next decision"
```

> 			label swap4 # swap middle and right

Reviewer 2: ```suggestion
			label "swap 4" # swap middle and right
```

			`	The dealer snaps the middle and right cards together, exchanging their places with a slick flick.`
				goto nextdecision
			label swap5 # swap left and middle
			`	The dealer flicks the left and middle cards across each other in a quick swap.`

> 				goto nextdecision

Reviewer 2: ```suggestion
				goto "next decision"
```

			label ending
			`	With a theatrical flourish, he fans his hands across the table, then leans in. "Where's the ace?"`
			choice
				`	"Left."`
					goto leftcard
				`	"Middle."`
					goto middlecard
				`	"Right."`
			action
				"monte: guess" = 2
			`	The dealer dramatically puts his hand on the rightmost card that you chose...`
				goto failedcard
			label leftcard
			action
				"monte: guess" = 0

> 			`	The dealer dramatically puts his hand on the leftmost card that you chose...`

Reviewer 2: ```suggestion
			`	The dealer dramatically puts his hand on the leftmost card...`
```

				goto failedcard

> 			label middlecard

Reviewer 2: ```suggestion
			label "middle card"
```

			action
				"monte: guess" = 1
			`	The dealer dramatically puts his hand on the middle card that you chose...`

> 			label failedcard

Reviewer 2: ```suggestion
			label "failed card"
```

			branch correct
				and
					"monte: card spot" == "monte: guess"
					random < "monte: not scam chance" + 50 - 25 * "monte: games played"
			`	And flips your chosen, decidedly non-ace card. You've lost 1,000 credits this round.`
			`	Then he playfully reveals the left card: the ace.`
				to display
					or
						and
							"monte: card spot" == 0
							"monte: card spot" != "monte: guess"
						and
							"monte: card spot" != 0
							"monte: card spot" == "monte: guess"
			`	With a single, clean motion he shows that the middle card was the ace.`
				to display
					or
						and
							"monte: card spot" == 1
							"monte: card spot" != "monte: guess"
						and
							"monte: card spot" != 1
							"monte: card spot" != 2
							"monte: card spot" == "monte: guess"
			`	He then sweeps his hand to reveal that the right card was the ace.`
				to display
					and
						"monte: card spot" == 2
						"monte: card spot" != "monte: guess"
			branch scammed
				"monte: card spot" == "monte: guess"
			`	"Better luck next time."`
				goto again
			label scammed
			action

> 				"monte: scammed" = 1

Reviewer 2: For conditions that only ever equal `0` and `1`, use `set` and `has` instead.
```suggestion
				set "monte: scammed"
```

			`	The dealer shrugs, "Better luck next time."`
				goto again
			label correct
			`	And flips over an ace! The dealer seems bewildered, but begrudgingly hands over 2,000 credits, doubling your money.`
			action
				payment 2000
			label again
			branch cutoff
				"monte: games played" < 3

> 			`	The dealer looks spooked by some unseen authorities and shoos you away, quickly packing up and disappearing into the spaceport.`

Reviewer 2: ```suggestion
			`	He prepares to lay out a new set of cards, only to suddenly shoo you away after appearing to be spooked by some unseen authorities. The dealer quickly packs up and disappears into the spaceport.`
```

				goto trueending
			label cutoff

> 			`	The dealer gives you a look. "Fancy another game?"`

Reviewer 2: Reduce repetition of "the dealer".
```suggestion
			`	Then he gives you a look. "Fancy another game?"`
```

			action
				"monte: games played" ++
			choice
				`	"Of course!"`
					to display
						and
							credits > 1000

> 					goto newgame

Reviewer 2: ```suggestion
					goto "new game"
```

				`	"You cheated!"`
					to display
						"monte: scammed" >= 1
				`	"Not today."`
					goto trueending
			branch pirateworld

> 				"flagship planet attribute: pirate" > 0

Reviewer 2: ```suggestion
				has "flagship planet attribute: pirate"
```

			`	The dealer feigns offense, shoulders lifting. "Cheated? Me? You've got quite the imagination, friend." He slides forward a fresh credit chip with a crooked grin. "Prove it. Same rules, double or nothing."`
			choice
				`	"Fine. But I'm watching you."`
					to display
							credits > 1000
					goto newgame
				`	"And get scammed again? I don't think so."`

> 					goto trueending

Reviewer 2: ```suggestion
					goto "true ending"
```

			label pirateworld
			`	The dealer throws back his head and laughs. "Cheated? Sure did, what're you gonna do about it, eh?"`
			`	From the crowd, several burly figures step forward, weapons on hips, while the dealer's grin never fades.`
			`	"Matter of fact, you've ruined the game. That's a mark's fee, 10,000 credits. Hand it over." He jerks a thumb at the hulking men circling close. "They'll collect if you're shy."`
			choice
				`	"You got it, here's 10,000 credits."`
					to display
						credits > 10000

> 				`	(Book it for your ship.)`

Reviewer 2: ```suggestion
				`	(Book it for my ship.)`
```

> 					goto bookit

Reviewer 2: ```suggestion
					goto "book it"
```

> 			`	Your credit chips thud as they land on the table, and you see the dealer's grin widen while you're fleeced and embarrassed.`

Reviewer 2: The credits are never actually taken away.

Author: Whoops, fixed.

> 				goto trueending

Reviewer 2: ```suggestion
				goto "true ending"
```

			label bookit
			action
				clear "monte: games played"
				clear "monte: multiplication"
				clear "monte: addition"
				clear "monte: card spot"
				clear "monte: guess"
				clear "monte: step"
				clear "monte: not scam chance"
				clear "monte: scammed"
				"reputation: Pirate" -= 2.5
			`	You bolt for the <ship>.`
			`	"After them!" the dealer shouts, while his men leap into pursuit, curses and boots echoing behind you. The sound of commotion behind you dissolves as you make it aboard your ship before they can catch up.`

> 				launch

Reviewer 2: This accepts the mission.
```suggestion
				flee
```

> 			label trueending

Reviewer 2: Labels should have spaces if needed.
```suggestion
			label "true ending"
```

phrase "three card monte opening"
	word
		"You step down from your ship onto the main concourse of the spaceport, "
		"Strolling past the smoky glow of the spaceport bar, you drift into the main area, "
		"After a quiet walk around the area, you find yourself back in the heart of the spaceport, "
		"You descend from your ship to enter the spaceport, "
	word
		"where a semicircle of onlookers has formed around a battered folding table and a trio of worn cards. "
		"and find your path blocked by a tight ring of locals listening as a nimble-fingered dealer swaps three cards across a board. "
		"when a sharp laugh draws your attention to a small card game on a crate. "
		"and see a tight cluster of faces leaning in around a makeshift table as quick hands shuffle three tired cards. "
	word
		"The dealer looks up and grins. "
		"The dealer glances up, eyebrows arched. "
		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you to step closer. "
		"The dealer finally notices you, and his grin widens. "

phrase "three card monte opening 2"
	word
		"He offers a simple challenge: "
		"He spreads his arms over the cards in invitation: "

> 		" "

Reviewer: Using a space here wouldn't work. I think you can just do this?
```suggestion
		""
```

		"He grandly gestures at the three waiting cards: "

mission "Three Card Monte"
	minor

> 	repeat

Reviewer: It's probably fine the way it is (although I'd like feedback from somebody else as well)

> 	repeat

Reviewer 3: It might be nice to have something like `<base chance of offering>/(<times accepted>+1)`, so that it becomes slightly less common if you keep repeating it. I'm not sure how that would work with integer division though.

> 	repeat

Reviewer 3: ~~Offer math was made to be over-engineered~~

The `random < 5 + 15 / <times offered>` seems like it should be a good way to do it (making sure to avoid division by 0, of course, I forget when offers are counted).

Of course, the other way also works, so if other people prefer it I'm fine either way.

> 	repeat

Author: Done! Tested and the mission did offer, so at least it doesn't break anything.

				goto "next decision"
			label "swap 4" # swap middle and right
			`	The dealer snaps the middle and right cards together, exchanging their places with a slick flick.`
				goto "next decision"
			label "swap 5" # swap left and middle
			`	The dealer flicks the left and middle cards across each other in a quick swap.`
				goto "next decision"
			label ending
			`	With a theatrical flourish, he fans his hands across the table, then leans in. "Where's the ace?"`
			choice
				`	"Left."`
					goto "left card"
				`	"Middle."`
					goto "middle card"
				`	"Right."`
			action
				"monte: guess" = 2
			`	The dealer dramatically puts his hand on the rightmost card...`
				goto "failed card"
			label "left card"
			action
				"monte: guess" = 0
			`	The dealer dramatically puts his hand on the leftmost card...`
				goto "failed card"
			label "middle card"
			action
				"monte: guess" = 1
			`	The dealer dramatically puts his hand on the middle card...`
			label "failed card"
			branch correct
				and
					"monte: card spot" == "monte: guess"
					random < "monte: not scam chance" - 25 * "monte: games played"

> 			`	And flips your chosen, decidedly non-ace card. You've lost 1,000 credits this round.`

Author: I love that idea, done.

			`	Then he playfully reveals the left card: the ace.`
				to display
					or
						and
							"monte: card spot" == 0
							"monte: card spot" != "monte: guess"
						and
							"monte: guess" != 0
							"monte: card spot" == "monte: guess"
			`	With a single, clean motion he shows that the middle card was the ace.`
				to display
					or
						and
							"monte: card spot" == 1
							"monte: card spot" != "monte: guess"
						and
							"monte: guess" == 0
							"monte: card spot" == "monte: guess"
			`	He then sweeps his hand to reveal that the right card was the ace.`
				to display
					"monte: card spot" == 2
					"monte: card spot" != "monte: guess"
			branch scammed
				"monte: card spot" == "monte: guess"
			`	"Better luck next time."`
				goto again
			label scammed
			action
				set "monte: scammed"

> 			`	The dealer shrugs, "Better luck next time."`

Reviewer: I think these should be the same, or people who have read the files will always know when they're being scammed.
```suggestion
			`	The dealer shrugs. "Better luck next time."`
				goto again
			label scammed
			action
				set "monte: scammed"
			`	The dealer shrugs. "Better luck next time."`
```

				goto again
			label correct
			`	And flips over an ace! The dealer seems bewildered, but begrudgingly hands over 2,000 credits, doubling your money.`
			action
				payment 2000
			label again
			branch cutoff
				"monte: games played" < 3
			`	He prepares to lay out a new set of cards, only to suddenly shoo you away after appearing to be spooked by some unseen authorities. The dealer quickly packs up and disappears into the spaceport.`
				goto "true ending"
			label cutoff
			`	Then he gives you a look. "Fancy another game?"`
			action
				"monte: games played" ++
			choice
				`	"Of course!"`
					to display
						and

> 							credits > 1000

Reviewer: ```suggestion
						credits > 1000
```

					goto "new game"
				`	"You cheated!"`
					to display
						has "monte: scammed"
				`	"Not today."`
					goto "true ending"
			branch "pirate world"
				has "flagship planet attribute: pirate"
			`	The dealer feigns offense, shoulders lifting. "Cheated? Me? You've got quite the imagination, friend." He slides forward a fresh credit chip with a crooked grin. "Prove it. Same rules, double or nothing."`
			choice
				`	"Fine. But I'm watching you."`
					to display
							credits > 1000

> 					goto "new game"

Reviewer: ```suggestion
					goto "new game start"
```

> 				`	"And get scammed again? I don't think so."`

Reviewer: ```suggestion
				`	"Sorry, but I don't have enough money."`
					goto "true ending"
					to display
						credits < 1000
				`	"And get scammed again? I don't think so."`
```

					goto "true ending"
			label "pirate world"
			`	The dealer throws back his head and laughs. "Cheated? Sure did, what're you gonna do about it, eh?"`
			`	From the crowd, several burly figures step forward, weapons on hips, while the dealer's grin never fades.`
			`	"Matter of fact, you've ruined the game. That's a mark's fee, 10,000 credits. Hand it over." He jerks a thumb at the hulking men circling close. "They'll collect if you're shy."`
			choice
				`	"You got it, here's 10,000 credits."`
					to display
						credits > 10000
				`	(Book it for my ship.)`
					goto "book it"
			action
				payment -10000
			`	Your credit chips thud as they land on the table, and you see the dealer's grin widen while you're fleeced and embarrassed.`
				goto "true ending"
			label "book it"
			action
				clear "monte: games played"
				clear "monte: multiplication"
				clear "monte: addition"
				clear "monte: card spot"
				clear "monte: guess"
				clear "monte: step"
				clear "monte: not scam chance"
				clear "monte: scammed"
				"reputation: Pirate" -= 2.5
			`	You bolt for the <ship>.`

> 			`	"After them!" the dealer shouts, while his men leap into pursuit, curses and boots echoing behind you. The sound of commotion behind you dissolves as you make it aboard your ship before they can catch up.`

Reviewer: ```suggestion
			`	"After them!" the dealer shouts, while his men leap into pursuit, their curses and boots echoing behind you. The sound of commotion behind you dissolves as you make it aboard your ship before they can catch up.`
```
I _think_ this improves it?

				clear "second place margin"
				clear "last place margin"
				clear "hovercraft racing: talked to fan"

phrase "three card monte opening"
	word
		"You step down from your ship onto the main concourse of the spaceport, "

> 		"Strolling past the smoky glow of the spaceport bar, you drift into the main area, "

Reviewer 3: "The main area" is a little awkward and unclear. Maybe something more like this would work better?

```suggestion
		"Strolling past the smoky glow of the spaceport bar, you head along the port's main street "
```

		"After a quiet walk around the area, you find yourself back in the heart of the spaceport, "
		"You descend from your ship to enter the spaceport, "
	word
		"where a semicircle of onlookers has formed around a battered folding table and a trio of worn cards. "
		"and find your path blocked by a tight ring of locals listening as a nimble-fingered dealer swaps three cards across a board. "
		"when a sharp laugh draws your attention to a small card game on a crate. "
		"and see a tight cluster of faces leaning in around a makeshift table as quick hands shuffle three tired cards. "
	word
		"The dealer looks up and grins. "
		"The dealer glances up, eyebrows arched. "
		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you to step closer. "
		"The dealer finally notices you, and his grin widens. "

phrase "three card monte opening 2"
	word
		"He offers an easy challenge: "
		"He spreads his arms over the cards in invitation: "
		""
		"He grandly gestures at the three waiting cards: "

phrase "card anything but an ace"
	word
		"king"
		"queen"
		"jack"
		"ten"

> 		"two"

Reviewer 3: Hmmm... in my mind, the King of Hearts is a proper noun and should be capitalized. I'm not sure what the standard is though, if other people disagree then this is fine as-is.

Reviewer 3: In that case, it's probably fine in lower case.

mission "Three Card Monte"
	minor
	repeat
	"offer precedence" 10
	to offer
		random < 20
		random < 2 + 5 * "flagship planet attribute: tourism" + 5 * "flagship planet attribute: urban" + 10 * "flagship planet attribute: pirate" - "Three Card Monte: declined"
		credits > 1000
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Pirate"
		not attributes "deep" "station"
	on offer
		conversation
			`${three card monte opening}`
			`	The dealer tips a weathered hat. "Always a pleasure to see another captain of fortune." His gaze never leaves your wallet.`
				to display
					"reputation: Pirate" > 100
			`	${three card monte opening 2}"The game's simple. Stake 1,000 credits and find the ace to double your money."`
			action
				"monte: not scam chance" = "reputation: Pirate" / 10 + 100
				"monte: not scam chance" >?= 60 # There's always at least a 60% of not being scammed first round, meaning the player always comes out ahead in the first game
			label "new game"
			`	The dealer waits expectantly.`
			choice
				`	(Put down 1,000 credits to play the game.)`
					to display
						credits > 1000
				`	(Pat my empty pockets. Can't play today.)`

> 					decline

Reviewer 3: It would also be neat to have a brief stub here where the scammer tells you what a shame it is you can't play or something.

					to display
						credits < 1000
				`	(Skip the game.)`
					decline
			label "new game start"
			action
				"monte: multiplication" = 1 + "roll: 2"
				"monte: addition" = "roll: 3"
				"monte: card spot" = ( "monte: multiplication" + "monte: addition" ) % 3
				"monte: step" = 1
				clear "monte: scammed"
				payment -1000

> 			`	You drop a 1,000 credit chip to play. "All you need to do is find the ace," the dealer says, flashing the ace of ${card suits} faceup towards you, then setting it down between two other facedown cards in the center. He starts shuffling with a practiced twist.`

Reviewer 3: Again, in my mind card names are proper nouns (and StackExchange seems to agree with me). I'd probably capitalize all of these?

Author: Ditto to above, happy either way if anyone feels strongly.

						has "monte: scammed"
				`	"Not today."`
					goto "true ending"
			branch "pirate world"
				has "flagship planet attribute: pirate"
			`	The dealer feigns offense, shoulders lifting. "Cheated? Me? You've got quite the imagination, friend." He slides forward a fresh credit chip with a crooked grin. "Prove it. Same rules, double or nothing."`
			choice
				`	"Fine. But I'm watching you."`
					to display
							credits > 1000
					goto "new game start"
				`	"Sorry, but I don't have enough money."`
					goto "true ending"
					to display
						credits < 1000
				`	"And get scammed again? I don't think so."`
					goto "true ending"
			label "pirate world"
			`	The dealer throws back his head and laughs. "Cheated? Sure did, what're you gonna do about it, eh?"`
			`	From the crowd, several burly figures step forward, weapons on hips, while the dealer's grin never fades.`
			`	"Matter of fact, you've ruined the game. That's a mark's fee, 10,000 credits. Hand it over." He jerks a thumb at the hulking men circling close. "They'll collect if you're shy."`
			choice
				`	"You got it, here's 10,000 credits."`
					to display
						credits > 10000
				`	(Book it for my ship.)`

> 					goto "book it"

Reviewer 3: Where's the option to reach for my gun?

It would be interesting to have an option to intimidate/fight the pirates that has a decent chance of success with sufficient crew as backup and is a death sentence if you're alone.

	word
		"hearts"
		"spades"
		"clubs"
		"diamonds"

mission "Three Card Monte"
	minor
	repeat
	"offer precedence" 10
	to offer
		random < 5 + 15 / "Three Card Monte: offered"
		random < 2 + 5 * "flagship planet attribute: tourism" + 5 * "flagship planet attribute: urban" + 10 * "flagship planet attribute: pirate" - "Three Card Monte: declined" + "monte: accepted game"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Pirate"
		not attributes "deep" "station"
	on offer
		conversation
			`${three card monte opening}`
			`	The dealer tips a weathered hat. "Always a pleasure to see another captain of fortune." His gaze never leaves your wallet.`
				to display
					"reputation: Pirate" > 100
			`	${three card monte opening 2}"The game's simple. Stake 1,000 credits and find the ace to double your money."`
			action
				"monte: not scam chance" = "reputation: Pirate" / 10 + 100
				"monte: not scam chance" >?= 60 # There's always at least a 60% of not being scammed first round, meaning the player always comes out ahead in the first game
			`	The dealer waits expectantly.`
			choice
				`	(Put down 1,000 credits to play the game.)`
					to display

> 						credits < 1000

Reviewer 4: Did you mean:
```suggestion
						credits > 1000
```

			label "swap 3" # swap counterclockwise
			`	The dealer shifts the trio one step to the left, the leftmost card ending up on the right.`
				goto "next decision"
			label "swap 4" # swap middle and right
			`	The dealer snaps the middle and right cards together, exchanging their places with a slick flick.`
				goto "next decision"
			label "swap 5" # swap left and middle
			`	The dealer flicks the left and middle cards across each other in a quick swap.`
				goto "next decision"
			label ending
			`	With a theatrical flourish, he fans his hands across the table, then leans in. "Where's the ace?"`
			choice
				`	"Left."`
					goto "left card"
				`	"Middle."`
					goto "middle card"
				`	"Right."`
			action
				"monte: guess" = 2
			`	The dealer dramatically puts his hand on the rightmost card...`
				goto "failed card"
			label "left card"
			action
				"monte: guess" = 0
			`	The dealer dramatically puts his hand on the leftmost card...`
				goto "failed card"
			label "middle card"
			action
				"monte: guess" = 1
			`	The dealer dramatically puts his hand on the middle card...`
			label "failed card"
			branch correct
				and
					"monte: card spot" == "monte: guess"

> 					random < "monte: not scam chance" - 25 * "monte: games played"

Reviewer 4: ```suggestion
				"monte: card spot" == "monte: guess"
				random < "monte: not scam chance" - 25 * "monte: games played"
```

			label again
			branch cutoff
				"monte: games played" < 3
			`	He prepares to lay out a new set of cards, only to suddenly shoo you away after appearing to be spooked by some unseen authorities. The dealer quickly packs up and disappears into the spaceport.`
				goto "true ending"
			label cutoff
			`	Then he gives you a look. "Fancy another game?"`
			action
				"monte: games played" ++
			choice
				`	"Of course!"`
					to display
						credits > 1000
					goto "new game start"
				`	"You cheated!"`
					to display
						has "monte: scammed"
				`	"Not today."`
					goto "true ending"
			branch "pirate world"
				has "flagship planet attribute: pirate"
			`	The dealer feigns offense, shoulders lifting. "Cheated? Me? You've got quite the imagination, friend." He slides forward a fresh credit chip with a crooked grin. "Prove it. Same rules, double or nothing."`
			choice
				`	"Fine. But I'm watching you."`
					to display
						credits > 1000
					goto "new game start"
				`	"Sorry, but I don't have enough money."`
					goto "true ending"
					to display
						credits < 1000
				`	"And get scammed again? I don't think so."`
					goto "true ending"
			label "pirate world"

> 			`	The dealer throws back his head and laughs. "Cheated? Sure did, what're you gonna do about it, eh?"`

Reviewer 5: The "what're you gonna do about it" works, as does people drawing weapons and threatening you, but I'm not sure the dealer would just outright admit it immediately.

				clear "second place margin"
				clear "last place margin"
				clear "hovercraft racing: talked to fan"

phrase "three card monte opening"
	word
		"You step down from your ship onto the main concourse of the spaceport, "
		"Strolling past the smoky glow of the spaceport bar, you head along the port's main street "
		"After a quiet walk around the area, you find yourself back in the heart of the spaceport, "
		"You descend from your ship to enter the spaceport, "
	word
		"where a semicircle of onlookers has formed around a battered folding table and a trio of worn cards. "
		"and find your path blocked by a tight ring of locals listening as a nimble-fingered dealer swaps three cards across a board. "
		"when a sharp laugh draws your attention to a small card game on a crate. "
		"and see a tight cluster of faces leaning in around a makeshift table as quick hands shuffle three tired cards. "
	word
		"The dealer looks up and grins. "
		"The dealer glances up, eyebrows arched. "
		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you to step closer. "

> 		"The dealer finally notices you, and his grin widens. "

Reviewer: ```suggestion
		"The dealer looks up and grins."
		"The dealer glances up, eyebrows arched."
		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you to step closer."
		"The dealer finally notices you, and his grin widens."
```

	word
		"king"
		"queen"
		"jack"
		"ten"
		"two"
	word
		" of"

phrase "card suits"
	word
		"hearts"
		"spades"
		"clubs"
		"diamonds"

mission "Three Card Monte"
	minor
	repeat
	"offer precedence" 10
	to offer
		random < 5 + 15 / "Three Card Monte: offered"
		random < 2 + 5 * "flagship planet attribute: tourism" + 5 * "flagship planet attribute: urban" + 10 * "flagship planet attribute: pirate" - "Three Card Monte: declined" + "monte: accepted game"
		credits > 1000
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Pirate"
		not attributes "deep" "station"
	on offer
		conversation
			`${three card monte opening}`
			`	The dealer tips a weathered hat. "Always a pleasure to see another captain of fortune." His gaze never leaves your wallet.`
				to display
					"reputation: Pirate" > 100
			`	${three card monte opening 2}"The game's simple. Stake 1,000 credits and find the ace to double your money."`
			action
				"monte: not scam chance" = "reputation: Pirate" / 10 + 100
				"monte: not scam chance" >?= 60 # There's always at least a 60% of not being scammed first round, meaning the player always comes out ahead in the first game

> 			label "new game"

Reviewer: ```suggestion
```
Nothing else leads to this label.

			choice
				`	"You got it, here's 10,000 credits."`
					to display
						credits > 10000
				`	(Book it for my ship.)`
					goto "book it"
				`	(Reach for your gun.)`
					goto gun
				`	(Whistle for your crew.)`
					goto whistle
					to display
						"flagship crew" > 1
			action
				payment -10000
			`	Your credit chips thud as they land on the table, and you see the dealer's grin widen while you're fleeced and embarrassed.`
				goto "true ending"
			label gun
			action
				clear "monte: games played"
				clear "monte: multiplication"
				clear "monte: addition"
				clear "monte: card spot"
				clear "monte: guess"
				clear "monte: step"
				clear "monte: not scam chance"
				clear "monte: scammed"
			branch "gun death"
				random < 80 - ( "flagship attribute: capture attack" / "flagship crew" ) / 50 # 20X average capture attack
			`	You draw and fire. The dealer's smirk vanishes - he staggers, then collapses. His men scatter, and you quickly make it back to your ship.`
				flee
			label "gun death"

> 			`	You move for your weapon but the pirates move first. Hot pain blooms in your chest, your hands go numb as your gun clatters away. You look down and see red, then nothing at all.`

Reviewer: ```suggestion
			`	You move for your weapon but the pirates move first. Hot pain blooms in your chest and your hands go numb as your gun clatters away. You look down and see red, then nothing at all.`
```
